src/style.css

```diff
@@ -0,0 +1,20 @@
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
+
+div {
```

Comment:
use classes for styling, don't style elements directly

Reply:
fixed